desktop_version/src/Graphics.cpp

```diff
@@ -3184,6 +3185,8 @@ void Graphics::screenshake(void)
     clear();
 
     copy_texture(menuoffTexture, NULL, NULL, 0, NULL, flipmode ? SDL_FLIP_VERTICAL : SDL_FLIP_NONE);
+
+    key.controllerRumble(0x7FFF,10);
```

Comment:
Please don't start rumble inside of rendering. Rendering should be rendering only.